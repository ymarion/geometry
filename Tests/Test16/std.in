# Min UNDO/REDO for this test: 5
# 1-2
C c 1 1 1
LIST
# 3-4
DELETE c
LIST
# 5-6
C c 2 2 2
LIST
# 7-8
DELETE c
LIST
# 9-10
C c 3 3 3
LIST
# 11-12
DELETE c
LIST
# 13-14
UNDO
LIST
# 15-16
UNDO
LIST
# 17-18
REDO
LIST
# 19-20
UNDO
LIST
# 21-22
UNDO
LIST
# 23-24
UNDO
LIST
# 25-26
UNDO
LIST
# Bye
EXIT
