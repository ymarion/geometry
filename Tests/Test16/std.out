# 1-2
OK
C c 1 1 1
# 3-4
OK
EMPTY
# 5-6
OK
C c 2 2 2
# 7-8
OK
EMPTY
# 9-10
OK
C c 3 3 3
# 11-12
OK
EMPTY
# 13-14
OK
C c 3 3 3
# 15-16
OK
EMPTY
# 17-18
OK
C c 3 3 3
# 19-20
OK
EMPTY
# 21-22
OK
C c 2 2 2
# 23-24
OK
EMPTY
# 25-26
OK
C c 1 1 1
